utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/AvroSchemaCreator.scala

```diff
@@ -0,0 +1,59 @@
+package pl.touk.nussknacker.engine.avro
+
+import org.apache.avro.{LogicalType, LogicalTypes, Schema}
+import org.apache.avro.Schema.{Field, Type}
+
+import scala.util.Random
+
+object AvroSchemaCreator {
```

Comment:
It shouldn't be in avro-component-utils. Why not just move to test classes?

Reply:
I thought this object will be helpfully in other tests, we don't have any avro-test-utils, so I decided to put it here :) But if you insist?